guide/english/linux/get-root-privileges-using-the-sudo-command/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Get root privileges using the sudo command.
+---
+
+
+# Get root privileges using the sudo command.
+
+The `sudo` (super do) command in Linux is used to give certain user(s) only a few super (root/administrator) permissions. And with those granted permissions a regular user can issue root commands to carry out any administrative tasks on the system.
+
+If you want to use `sudo` to run commands as a regular user, you’d prepend the `sudo` command to the command you intend to execute using this format:
+
+`sudo <command>`
+
+For example;
+
+`sudo` 
```

Comment:
Please provide an example here (believe you've removed a full example that was in your original article)